Instagram.xcworkspace/xcuserdata/sarahlywang.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Interesting. Not sure why this got checked in. Would have expected it to be gitignored.